src/common/classes/vector.h

```diff
@@ -118,6 +126,20 @@ class Vector
 		return data[count];
 	}
 
+	void push(const T* items, const FB_SIZE_T itemsSize)
+	{
+		fb_assert(count <= FB_MAX_SIZEOF - itemsSize);
+		fb_assert(count + itemsSize <= Capacity);
+		memcpy(data + count, items, sizeof(T) * itemsSize);
+		count += itemsSize;
+	}
+
+	void append(const T* items, const FB_SIZE_T itemsSize)
```

Comment:
@AlexPeshkoff I think to identical methods with different names is not a good thing. It's better to stablish on one or another. Also, itemsCount would be a better parameter name than itemsSize. When I read, I thought itemsSize was something about sizeof a single items buffer.